CMakeLists.txt

```diff
@@ -468,6 +467,7 @@ OCV_OPTION(WITH_ONNX "Include Microsoft ONNX Runtime support" OFF
 OCV_OPTION(WITH_TIMVX "Include Tim-VX support" OFF
   VISIBLE_IF TRUE
   VERIFY HAVE_TIMVX)
+# attention: Astra2, Gemini2, and Gemini2L cameras currently only support Windows and Linux kernel versions <= 4.15, and higher versions of Linux kernel may have exceptions.
```

Comment:
The statement about kernel version is not clear. Do you mean, that camera requires kernel 4.15 or older?
Please add reference to the camera documentation, ticket or whatever to track support status.